### ISSUE TITLE
ETA for moving repo to cloudfoundry-incubator or cloudfoundry org?

### ISSUE THREAD

**Issue by drnic** [pivotal/credhub-release#21]
I saw a new credhub 1.8.2 release in #bosh channel and went out trying to find the credhub-release repo but it wasn't under neither https://github.com/cloudfoundry nor https://github.com/cloudfoundry-incubator. I think Credhub's been in the incubator for a year now. When will it move to one of these two orgs?

**Comment by athornton2012** [comment]
@drnic I believe that it has something to do with the fact that we support Luna HSMs by including their .so and .jar files in CredHub... @crawsible  correct me if I am wrong! If this is the case, I imagine that we won't be able to move to the cloudfoundry org until we have completed a track of work to extract that into a separate repo.

**Comment by crawsible** [comment]
That's correct. We should no longer be blocked from moving to CFF after we've extracted the 3rd party libraries we've included in this repo by agreement with the third party.

**Comment by gberche-orange** [comment]
@crawsible @athornton2012 It seems the issue was closed by mistake as the repo is still hosted in the https://github.com/pivotal-cf/ organization. Can you please reopen the issue, or should I instead open a new one ?

**Comment by ankeesler** [comment]
We can reopen it. Our understanding is that we would need to work with the CF Extensions organization to get it moved over. Is that correct?

**Comment by hsinn0** [comment]
Closing old non-active issues.
